airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -832,6 +836,27 @@
         }
     }
 
+    function showMediaSessionMetadata(song) {
+        if ('mediaSession' in top.playQueue.navigator) {
+            var metadata = new MediaMetadata({
+                title: song.title,
+                artist: song.artist,
+                album: song.album,
+                artwork: [
+                    // Adding the '&size=' argument will not work here because
+                    // the JWT token used to expose this URL requires all
+                    // parameters to stay exactly the same.
+                    { src: song.remoteCoverArtUrl, type: 'image/jpeg' },
```

Comment:
two things here:
- you're not guaranteed a jpeg here, are you? it could be a png...
- why access the remote url, when this user is local? you don't need to go through the jwt, you've already authed this user

Reply:
The cover art controller always creates a JPEG when asked for resized cover art.
Good point re:using the local cover art.